### PR TITLE
Release notes for 2021.09.4 

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Adding release notes for 2021.09.4
- **Is this PR related to a Monolith release?** Yes, 2021.09.4

### Resolves:

* [OKTA-430690](https://oktainc.atlassian.net/browse/OKTA-430690)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/scopes/index.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ The following table shows the scopes that are currently available:
 | `okta.clients.manage`    | Allows the app to manage all OAuth/OIDC clients and to create new clients| [Dynamic Client Registration API](/docs/reference/api/oauth-clients/)|
 | `okta.clients.read`      | Allows the app to read information for all OAuth/OIDC clients           | [Dynamic Client Registration API](/docs/reference/api/oauth-clients/)|
 | `okta.clients.register`  | Allows the app to register (create) new OAuth/OIDC clients (but not read information about existing clients)| [Dynamic Client Registration API](/docs/reference/api/oauth-clients/#register-new-client) |
+| `okta.devices.manage`		| Allows the app to manage any device's profile      | [Devices API](/docs/reference/api/devices) |
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/scopes/index.md:20]:
```suggestion
| `okta.devices.manage`		| Allows the app to manage any Device's profile     | [Devices API](/docs/reference/api/devices/) |
```

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/scopes/index.md [hunk 2]
```diff
@@ -17,6 +17,8 @@ The following table shows the scopes that are currently available:
...
+| `okta.devices.read`		| Allows the app to read any device's profile      | [Devices API](/docs/reference/api/devices) |
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/scopes/index.md:21]:
```suggestion
| `okta.devices.read`		| Allows the app to read any Device's profile      | [Devices API](/docs/reference/api/devices/) |
```

File: packages/@okta/vuepress-site/docs/reference/api/devices/index.md [hunk 3]
```diff
@@ -0,0 +1,855 @@
...
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/devices"
+```
+
+##### Response
+
+```json
+HTTP/1.1 200 OK
+Content-Type: application/json
+Link: <https://{yourOktaDomain}/api/v1/devices?limit=200>; rel="self"
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/devices/index.md:225]: I don't know why `{yourOktaDomain}`is flagged as red here?

**Review by vanngo-okta — COMMENTED** [review]
A few questions regarding the Devices API

File: packages/@okta/vuepress-site/docs/reference/core-okta-api/index.md [hunk 4]
```diff
@@ -25,6 +25,7 @@ REST endpoints to configure objects whenever you need. For example:
 - The [Sessions API](/docs/reference/api/sessions/) creates and manages user's authentication sessions.
 - The [Policy API](/docs/reference/api/policy/) creates and manages settings such as a user's session lifetime.
 - The [Factors API](/docs/reference/api/factors/) is used to enroll, manage, and verify factors for multi-factor authentication (MFA).
+- The [Devices API](/docs/reference/api/devices) is used to manage device identity and lifecycle.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/core-okta-api/index.md:28]:
```suggestion
- The [Devices API](/docs/reference/api/devices) is used to manage Device identity and lifecycle.
```

File: packages/@okta/vuepress-site/docs/reference/api/devices/index.md (hunk 3 above)
Comment (apoorvadeshpande-okta) [reply packages/@okta/vuepress-site/docs/reference/api/devices/index.md:225]: 1. This is okay to be RED as it's due to presence of tags `<>`
2. We should only add `${yourOktaDomain}` variable when it needs to be replaced by the developer when making an API call
3. Everything in the response is a placeholder for what server returns and here it represents developers okta domain.

**Review by apoorvadeshpande-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/reference/api/devices/index.md [hunk 5]
```diff
@@ -0,0 +1,855 @@
...
+
+The enrollment flow of the Okta Verify creates a device in the Okta Device Inventory. [Device Registration](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/devices/device-registration.htm)
+
+### Get a Device by ID
+
+<ApiOperation method="get" url="/api/v1/devices/${deviceId}" />
+
+Fetches a Device by its `id`. If you don't know the `id`, you can [List Devices](#list-devices).
+
+#### Permitted OAuth2 scopes
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/devices/index.md:50]:
```suggestion
#### Permitted OAuth 2.0 scopes 
```

File: packages/@okta/vuepress-site/docs/reference/api/devices/index.md [hunk 6]
```diff
@@ -0,0 +1,855 @@
...
+| Search term example                             | Description                                      |
+| :---------------------------------------------- | :----------------------------------------------- |
+| `status eq "ACTIVE"`                            | Devices that have a `status` of `ACTIVE`         |
+| `lastUpdated gt "yyyy-MM-dd'T'HH:mm:ss.SSSZ"`   | Devices last updated after a specific timestamp  |
+| `id eq "guo4a5u7JHHhjXrMK0g4"`                  | Devices with a specified `id`                    |
+| `profile.displayName eq "Bob"`                  | Devices that have a `displayName` of `Bob`       |
+| `profile.platform eq "WINDOWS"`                 | Devices that have an `platform` of `WINDOWS`     |
+| `profile.sid sw "S-1" `                         | Devices whose `sid` starts with `S-1`            |
+
+#### Permitted OAuth2 scopes
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/devices/index.md:180]:
```suggestion
#### Permitted OAuth 2.0 scopes 
```

File: packages/@okta/vuepress-site/docs/reference/api/devices/index.md [hunk 7]
```diff
@@ -0,0 +1,855 @@
...
+
+### Delete Device
+
+<ApiOperation method="delete" url="/api/v1/devices/${deviceId}" />
+
+Permanently deletes a Device that is in `DEACTIVATED` status. The Device can be transitioned to `DEACTIVATED` status using [deactivate](#deactivate-device) API.
+
+This deletion is destructive and deletes all the profile data related to the device. Once deleted, device data can't be recovered. A Device that is not in a `DEACTIVATED` state raises an error if Delete operation is attempted.
+
+#### Permitted OAuth2 scopes
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/devices/index.md:400]:
```suggestion
#### Permitted OAuth 2.0 scopes 
```

File: packages/@okta/vuepress-site/docs/reference/api/devices/index.md [hunk 8]
```diff
@@ -0,0 +1,855 @@
...
+
+### Activate Device
+
+<ApiOperation method="post" url="/api/v1/devices/${deviceId}/lifecycle/activate" />
+
+Sets a Device's `status` to `ACTIVE`.
+
+Activated devices can be used to create and delete Device User links.
+
+#### Permitted OAuth2 scopes
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/devices/index.md:460]:
```suggestion
#### Permitted OAuth 2.0 scopes 
```

File: packages/@okta/vuepress-site/docs/reference/api/devices/index.md [hunk 9]
```diff
@@ -0,0 +1,855 @@
...
+
+Sets a Device's `status` to `DEACTIVATED`. Deactivation causes a Device to lose all Device User links. A Device should be in `DEACTIVATED` status before it can be [deleted](#delete-device).
+
+> **Important**: 
+> 1. Deactivating a Device is a **destructive** operation. 
+> 2. This action cannot be reversed. 
+> 3. Device deactivation renders associated assets—such as device factors and management certificates—non-usable. Device reenrollment via Okta Verify is required to make such assets functional again.
+
+
+#### Permitted OAuth2 scopes
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/devices/index.md:517]:
```suggestion
#### Permitted OAuth 2.0 scopes 
```

File: packages/@okta/vuepress-site/docs/reference/api/devices/index.md [hunk 10]
```diff
@@ -0,0 +1,855 @@
...
+
+A device in `ACTIVE` status can transition to `SUSPENDED`. This status is meant to be temporary and hence not destructive in nature.
+
+Suspended devices:
+
+* Can be used to create and delete device user links.
+* Can only be [unsuspended](#unsuspend-device) or [deactivated](#deactivate-device).
+
+
+#### Permitted OAuth2 scopes
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/devices/index.md:576]:
```suggestion
#### Permitted OAuth 2.0 scopes 
```

File: packages/@okta/vuepress-site/docs/reference/api/devices/index.md [hunk 11]
```diff
@@ -0,0 +1,855 @@
...
+
+### Unsuspend Device
+
+<ApiOperation method="post" url="/api/v1/devices/${deviceId}/lifecycle/unsuspend" />
+
+Unsuspends a Device and by returning its `status` value to `ACTIVE`.
+
+This operation can only be performed on a Device that is in `SUSPENDED` status.
+
+#### Permitted OAuth2 scopes
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/devices/index.md:629]:
```suggestion
#### Permitted OAuth 2.0 scopes 
```

**Review by brentschaus-okta — APPROVED** [review]
LGTM!

**Pull request merged**